package.json

```diff
@@ -151,7 +151,7 @@
     "intl": "^1.2.5",
     "intl-messageformat": "^4.4.0",
     "intl-pluralrules": "^1.0.3",
-    "jpeg-js": "0.1.2",
+    "jpeg-js": "^0.4.1",
```

Comment:
we missed quite a few versions in there :)

Reply:
0.4.0 was the only big one I finally pushed recently, so you didn't miss much :)